jekyll/_cci2/workflows.md

```diff
@@ -172,21 +172,24 @@ Following is a screenshot of the Approval dialog box that appears when you click
 
 ## Scheduling a Workflow
 
-Workflows that are resource-intensive or that generate reports may be run on a schedule rather than on every commit. This feature is configured by adding a scheduled trigger to the configuration of the workflow. 
```

Comment:
No need to remove this information, please add back

Reply:
This information is incorrect and misleading:

- **Any** workflow can be scheduled — not just ones that are resource-intensive or that generate reports.
- It's not exactly right to call it a "scheduled trigger". In this case, the defined `schedule` **is** the trigger.

Reply:
I disagree, please add it back, this ticket was to add a note about a required key, not to go through and re-do this work (sorry, we just have too much work to do). Thank you!